03.asynchronous/async_await.js

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
+
+import timers from "timers/promises";
+import { initDb, run, get, all, closeDb } from "./promise.js";
+
+// async / await エラーなし
```

Comment:
INSERT の内容と表記を揃えたほうがよいと思います。

```suggestion
// async/await エラーなし
```

Reply:
たしかに揃えたほうがきれいですね！修正しました。